src/app/features/search/single-search-result/single-search-result.component.ts

```diff
@@ -29,8 +29,8 @@ export class SingleSearchResultComponent implements OnInit, OnDestroy {
             vin: this.vehicleTechRecord.vin,
             vrm: this.vehicleTechRecord.vrms.find(vrm => vrm.isPrimary)?.vrm,
             trailerId: this.vehicleTechRecord.trailerId,
-            make: record?.chassisMake,
-            model: record?.chassisModel,
+            make: (record?.vehicleType == 'hgv' ? record?.make : record?.chassisMake) ?? 'Make Not Found',
+            model: (record?.vehicleType == 'hgv' ? record?.model : record?.chassisModel) ?? 'Model Not Found',
```

Comment:
Since there is no specific AC or service design (at my knowledge) for this, I would keep the "empty" field as is with a `-` in the field